correction_x: 0  # given by hand if necessary
correction_y: 0  # given by hand if necessary
correction_z: 2.00868
correction_roll: 0.5859833
correction_pitch: 4.146606
correction_yaw: 0
